qemu.nsi

```diff
@@ -69,6 +69,46 @@ InstallDirRegKey HKLM "Software\qemu32" "Install_Dir"
 ; Request administrator privileges for Windows Vista.
 RequestExecutionLevel admin
 
+;--------------------------------
+; define language strings
+LangString QEMU_Wiki_online_Text ${LANG_English} "Visit the QEMU Wiki online!"
+LangString QEMU_Wiki_online_Text ${LANG_Italian} "Visit the QEMU Wiki online!"
```

Comment:
Did you miss this translation intentionally?

Reply:
OK. You could revsie as

LangString QEMU_Wiki_online_Text ${LANG_Italian} "Visita Wiki EMU su internet!"